OlawaleJoseph/BANKA
# #165265670 Install app dependencies

#### What does this PR do?
Installs the dependencies needed to build the application
#### Description of Task to be completed?
Installs packages that the application needs to function as expected
#### How should this be manually tested?
After cloning the repo, open package.json file to check the list of dependencies
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#165265670
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A
